reference/artifacts/types/docker-image/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: single
+title:  "Docker Image"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Docker image artifacts are stored in
+[registries](https://docs.docker.com/registry/), for example
+[GCR](https://gcr.io), or [Docker Hub](https://index.docker.io). They can be
+deployed to Kubernetes or App Engine, and generally trigger pipelines from
+notifications sent by their registry.
+
+## Fields
+
+| Field | Explanation |
+|-|-----------|
+| `type` | Always `docker/image`. |
+| `reference` | The fully-qualified image name, including version and registry. `gcr.io/project/my-image:v1.2` is an example. This can be done either by tag or image digest. |
```

Comment:
And this example is the former, right? Would it work to put this example in parens after "by tag" and and example digest in parens after "or image digest"?

Reply:
Good idea